lib/recipes/import/food_and_wine.rb

```diff
@@ -15,54 +15,32 @@ def recipe_total_time
         recipe_details['total_time']
       end
 
-      def recipe_ingredients # rubocop:disable Metrics
+      def recipe_ingredients
         document
-          .css('div.mntl-structured-ingredients')
-          .children
-          .reduce('') do |text, node|
-            next text unless node.element?
-
-            if node.name == 'ul'
-              text << node.children.map { |e| e.text.strip }.compact_blank.join("\n")
-            elsif node.name == 'p' && node.text.present?
-              text << "\n\n" if text.present?
-
-              text << "#{node.text.upcase}\n"
-            end
-
-            text
-          end
+          .css('ul.mm-recipes-structured-ingredients__list li p > span')
+          .map { |t| t.text.strip }
+          .compact_blank
+          .join("\n")
       end
 
-      def recipe_directions # rubocop:disable Metrics
+      def recipe_directions
         document
-          .css('div.recipe__steps-content')
-          .children
-          .reduce('') do |text, node|
-            next text unless node.element?
-
-            if node.name == 'ol'
-              text << node.children.map { |e| e.text.strip }.compact_blank.join("\n\n")
```

Comment:
This should work:
`text << node.children.map { |e| e.css('> p').text.strip }.compact_blank.join("\n\n")`